airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -219,7 +239,7 @@ const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessag
                   >
                     <FormattedMessage id="form.cancel" />
                   </Button>
-                  <Button type="submit" disabled={isSubmitting || !dirty || !isValid} isLoading={isSubmitting}>
+                  <Button type="submit" disabled={isSubmitting || !dirty} isLoading={isSubmitting}>
```

Comment:
I'm not a fan of making this button disabled, and also not giving users any visual hints what they need to do to enable it.  Willing to be persuaded otherwise, but I think this is better.